demisto_sdk/commands/common/tests/readme_test.py

```diff
@@ -82,3 +83,32 @@ def test_is_image_path_valid():
     assert images_paths[0] and alternative_images_paths[0] in captured_output.getvalue()
     assert images_paths[1] and alternative_images_paths[1] in captured_output.getvalue()
     assert images_paths[2] not in captured_output.getvalue()
+
+
+def test_verify_no_empty_sections():
```

Comment:
please add a happy/sad test case for each verification.
also, use the `TestSuite` instead of adding files, @jochman will be happy to assist upon request.